addons/l10n_cl/models/l10n_latam_document_type.py

```diff
@@ -13,7 +13,10 @@ class L10nLatamDocumentType(models.Model):
             ('invoice_in', 'Purchase Invoices'),
             ('debit_note', 'Debit Notes'),
             ('credit_note', 'Credit Notes'),
-            ('receipt_invoice', 'Receipt Invoice')])
+            ('receipt_invoice', 'Receipt Invoice'),
+            ('stock_picking', 'Stock Delivery')])
```

Comment:
```suggestion
            ('stock_picking', 'Stock Delivery'),
        ],
    )
```

To avoid useless diff like this when adding/removing options/parameters.